### ISSUE TITLE
<script ...> cannot have language= attribute, should be type=...

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#15]
<script ...> cannot have language= attribute, should be type=, in toggle language mod
==> Replace language=”...”  with type=”...”
